control/responsible_role.go

```diff
@@ -19,7 +19,7 @@ func findResponsibleRole(ct *Table) (*responsibleRole, error) {
 	}
 	// Not sure why we have to get the parent's parent, but we need to.
 	// If we only go up once, it won't find the other text nodes.
-	parentNode := nodes[0].Parent().Parent()
+	parentNode := nodes[0].Parent()
```

Comment:
Is this still necessary? Wouldn't the table cell's parent be the table itself? Also, the comment above this line needs to be updated.